be/src/vec/exec/join/process_hash_table_probe_impl.h

```diff
@@ -258,7 +259,7 @@ Status ProcessHashTableProbe<JoinOpType, Parent>::do_process(HashTableType& hash
               need_null_map_for_probe &&
                       ignore_null > (hash_table_ctx.keys, hash_table_ctx.bucket_nums.data(),
                                      probe_index, build_index, probe_rows, _probe_indexs.data(),
-                                     _probe_visited, _build_indexs.data());
+                                     _probe_visited, _build_indexs.data(), has_mark_join_conjunct);
```

Comment:
need some regression test

Reply:
There are some cases already covered this code in `regression-test/suites/nereids_p0/join/test_mark_join.groovy`, `regression-test/suites/nereids_p0/subquery/subquery_unnesting.groovy`.